diff --git a/node_modules/react-native-smooth-slider/src/Slider.js b/node_modules/react-native-smooth-slider/src/Slider.js
index fbce285..43de294 100644
--- a/node_modules/react-native-smooth-slider/src/Slider.js
+++ b/node_modules/react-native-smooth-slider/src/Slider.js
@@ -17,7 +17,7 @@ import {
     PanGestureHandler,
     State,
 } from 'react-native-gesture-handler';
-
+import  { ImagePropTypes} from 'deprecated-react-native-prop-types';
 import PropTypes from 'prop-types';
 
 var TRACK_SIZE = 4;
@@ -171,7 +171,7 @@ export default class Slider extends PureComponent {
         /**
          * Sets an image for the track.
          */
-        trackImage: Image.propTypes.source,
+        trackImage: ImagePropTypes.source,
 
         /**
          * The style applied to the thumb.
@@ -181,7 +181,7 @@ export default class Slider extends PureComponent {
         /**
          * Sets an image for the thumb.
          */
-        thumbImage: Image.propTypes.source,
+        thumbImage: ImagePropTypes.source,
 
         /**
          * Set this to true to be a vertical slider.
