diff --git a/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js b/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js
index 54cf4b9..b03883b 100644
--- a/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js
+++ b/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js
@@ -4,6 +4,7 @@ import { Text } from 'react-native';
 
 import RaisedButton from '../raised-button';
 import { styles } from './styles';
+import  { TextPropTypes} from 'deprecated-react-native-prop-types';
 
 export default class RaisedTextButton extends PureComponent {
   static defaultProps = {
@@ -16,7 +17,7 @@ export default class RaisedTextButton extends PureComponent {
 
     title: PropTypes.string.isRequired,
     titleColor: PropTypes.string,
-    titleStyle: Text.propTypes.style,
+    titleStyle: TextPropTypes.style,
     disabledTitleColor: PropTypes.string,
   };
 
diff --git a/node_modules/react-native-material-buttons/src/components/text-button/index.js b/node_modules/react-native-material-buttons/src/components/text-button/index.js
index 8b26f47..af4de25 100644
--- a/node_modules/react-native-material-buttons/src/components/text-button/index.js
+++ b/node_modules/react-native-material-buttons/src/components/text-button/index.js
@@ -4,6 +4,7 @@ import { Text } from 'react-native';
 
 import Button from '../button';
 import { styles } from './styles';
+import  { TextPropTypes} from 'deprecated-react-native-prop-types';
 
 export default class TextButton extends PureComponent {
   static defaultProps = {
@@ -21,7 +22,7 @@ export default class TextButton extends PureComponent {
 
     title: PropTypes.string.isRequired,
     titleColor: PropTypes.string,
-    titleStyle: Text.propTypes.style,
+    titleStyle: TextPropTypes.style,
     disabledTitleColor: PropTypes.string,
   };
 
